discover/commerce/articles/01-catalog-management/02-products/00-product-types-intro.markdown

```diff
@@ -0,0 +1,111 @@
+# Product Types [](id=product-types)
+
+The catalog supports three product types: simple, grouped and virtual.
+
+**Simple Products:** A simple product is any physical product for which
+inventory is tracked. Almost any physical item can be a simple product.
+
+**Grouped Products:** A grouped product consists of two or more products from
+the catalog, packaged together to be sold as a single item.  Grouped products
+are not tracked in inventory--instead, the catalog tracks all of the products
+contained in the group individually. A grouped product can contain simple
+products, other grouped products, or virtual products.
+
+**Virtual Products:** A virtual product is any non-physical product. Typically,
+a virtual product's inventory is not tracked. This may mean that the product is
+downloaded rather than shipped, as in the case of a music file or an electronic
+publication. Services, warranties and other goods that might be neither shipped
+nor downloaded would also be virtual products.
+
+## Creating a Product [](id=creating-a-product)
+
+To create a product:
+
+1.  From the catalog's *Products* tab, click on the
+    ![Add](../../../images/icon-add.png) button at the top-right of the
+    screen and then select the type of product you wish to create: simple,
+    grouped or virtual.
+
+2.  Next, you'll be prompted to enter basic product information into the *Add
+    Product* form (see below for details).
+
+    Once the *Add Product* form is complete to your satisfaction, click the
+    *Publish* button to add the product to the catalog. The product tab bar appears
+    at the top of the screen (and can be reached again at any time by navigating to
+    the catalog and selecting a product). The first tab, *Details*, is populated
+    with your responses from the *Add Product* form.
+
+3.  Enter additional information specific to the product type you selected in
+    step 1.
+
+    **For a Simple Product**, there is no additional information to enter. When
+    you create simple product, the standard product tab bar is displayed,
```

Comment:
Missing "a": "When you create **a** simple product;" change "is displayed" to "appears"